content/millennium/r4/encounters/encounter.md

```diff
@@ -62,6 +63,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  ID                                                        | Value\[x] Type                                                   | Description
 -----------------------------------------------------------|------------------------------------------------------------------|----------------------------------------------------------------
  `client-organization`                                     | [`Reference`](https://hl7.org/fhir/r4/references.html#Reference) | The financially responsible organization.
+ `custom-attribute`                                        | None (contains nested extensions)                                | A client defined custom attribute for the resource. Attribute values can be of type [`integer`], [`string`], [`CodeableConcept`], [`dateTime`].
```

Comment:
Nit: Should we have this be 'integer', 'string', 'CodeableConcept', **or** 'dateTime'?

Reply:
[Updated](https://github.com/cerner/fhir.cerner.com/pull/471/commits/e432dc96ef0915abfcadae4b5dbcb35f72b585e1)